Resources/Prototypes/Floof/Research/civilianservices.yml

```
@@ -1,3 +1,19 @@
+# Tier 1
+
+- type: technology   #this would match where the Ore bag of holding is at. a t1  7.5k research. if people add much more to hydroponics best to move it to t2
+  id: advanceHydroponics
```

Also this, capitalization and spelling:
```suggestion
  id: AdvancedHydroponics
```